.github/workflows/branch-build.yaml

```diff
@@ -37,12 +32,14 @@ jobs:
           cache-dependency-path: '**/package-lock.json'
 
       - name: Install dependencies
-        run: npm install
+        run: |
+          npm install
+          npm run bootstrap
 
       - name: Lint code
```

Comment:
Should we run not only linting but also executing the tests? (even if there are no test files yet)

Reply:
Good point 👍